src/Microsoft.ML.Data/DataLoadSave/Text/TextLoader.cs

```diff
@@ -427,9 +427,12 @@ internal bool TryUnparse(StringBuilder sb)
         public class Options
         {
             /// <summary>
-            /// Whether the input may include quoted values, which can contain separator characters, colons,
-            /// and distinguish empty values from missing values. When true, consecutive separators denote a
-            /// missing value and an empty value is denoted by \"\". When false, consecutive separators denote an empty value.
+            /// Whether the input may include double-quoted values. This parameter is used to distinguish separator characters
+            /// in an input value from actual separators. When true, separators within double quotes are treated as part of the
+            /// input value. When false, all separators, even those whitin quotes, are treated as delimiting a new column.
+            /// It is also used to distinguish empty values from missing values. When true, missing value are denoted by consecutive
```

Comment:
>It is also used to distinguish empty values from missing values [](start = 16, length = 63)

Is this correct? I know it used to be the case for text input, when the text type used to have a separate notion of "missing" and "empty", but not sure what the text loader does now in this case.

Reply:
No, this is no longer the case. Verified with a small text dataset. Behavior is the same when we have "" or consecutive separators, whaterver the value of `allowQuoting`.